src/platforms/python/guides/rq/index.mdx

```diff
@@ -34,45 +41,102 @@ Start your worker with:
 ```shell
 rq worker \
     -c mysettings \  # module name of mysettings.py
-    --sentry-dsn=""  # only necessary for RQ < 1.0
+    --sentry-dsn="___PUBLIC_DSN___"  # only necessary for RQ < 1.0
 ```
 
 The integration will automatically report errors from all RQ jobs.
 
 Generally, make sure that the **call to `init` is loaded on worker startup**, and not only in the module where your jobs are defined. Otherwise, the initialization happens too late and events might end up not being reported.
 
-In addition, make sure that **`init` is called only once** in your app. For example, if you have a Flask app and a worker that depends on the app, we recommend initializing Sentry with a single configuration that is suitable for Flask and RQ, as in:
+In addition, make sure that **`init` is called only once** in your app. For example, if you have a `Flask` app and a worker that depends on the app, we recommend initializing Sentry only once. (As the Flask integrations is also auto enabled you do not need to change the configuration shown above).
```

Comment:
```suggestion
In addition, make sure that **`init` is called only once** in your app. For example, if you have a `Flask` app and a worker that depends on the app, we recommend only initializing Sentry once. Note that because the Flask integration is enabled automatically, you don't need to change the configuration shown above.
```